### PR TITLE
review - party relationships

### PR DESCRIPTION
**Pull request by KellyVance**
Review of party/agent relationships
This is Edition 1 and does _not_ need to be a comprehensive list.
Current requirement is to cover the Business relationships we currently have, such as parent companies and subsidiaries . Additional classes can be added later as they become relevant. The extra concepts have been added for illustrative purposes to convey the intent and scope to future users and editors.

### CODE REVIEW

File: vocabularies/party-relationship.ttl [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pyrl: <http://linked.data.gov.au/def/party-relationship/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
```
Comment (DavidCrosswellGSQ) [inline vocabularies/party-relationship.ttl]: Need to add in @prefix sdo: <https://schema.org/> .

File: vocabularies/party-relationship.ttl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pyrl: <http://linked.data.gov.au/def/party-relationship/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/party-relationship> a owl:Ontology, skos:ConceptScheme .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/party-relationship.ttl]: end with a semi-colon

File: vocabularies/party-relationship.ttl [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    dct:created "2020-03-24"^^xsd:date ;
+    dct:creator <http://linked.data.gov.au/org/gsq>
```
Comment (DavidCrosswellGSQ) [inline vocabularies/party-relationship.ttl]: end with a semi-colon

File: vocabularies/party-relationship.ttl [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        pyrl:joint-venture,
+        pyrl:owner,
+        pyrl:parent,
+        pyrl:subsidiary,
+        pyrl:volunteer ;
+    skos:prefLabel "Party Relationship"@en .
+
+<http://linked.data.gov.au/org/gsq>
+    a sdo:Organization ;
+    sdo:name "Geological Survey of Queensland" ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/party-relationship.ttl]: End with full stop

File: vocabularies/party-relationship.ttl [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/party-relationship.ttl]: Remove this full stop

File: vocabularies/party-relationship.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: done.

File: vocabularies/party-relationship.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: done

File: vocabularies/party-relationship.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: done

File: vocabularies/party-relationship.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: Copied as per Nick Cars bulk vocabulary update

<http://linked.data.gov.au/org/gsq>
    a sdo:Organization ;
    sdo:name "Geological Survey of Queensland" ;
.

File: vocabularies/party-relationship.ttl (hunk 5 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: see above

File: vocabularies/party-relationship.ttl (hunk 1 above)
Comment (LukeHauck) [reply vocabularies/party-relationship.ttl]: Does this line need the <> brackets for correct syntax?

@prefix sdo: <https://schema.org/> .

File: vocabularies/party-relationship.ttl (hunk 4 above)
Comment (DavidCrosswellGSQ) [reply vocabularies/party-relationship.ttl]: While that syntax works, Nick should clean his code. The semi-colon is used to copy the subject to the next line.

File: vocabularies/party-relationship.ttl (hunk 1 above)
Comment (DavidCrosswellGSQ) [reply vocabularies/party-relationship.ttl]: Needs < > 
Github stripped these.

Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: Pesky. Changed.

File: vocabularies/party-relationship.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/party-relationship.ttl]: removed last semi-colon, moved fullstop up

**Pull request merged**